assets/performanceprofile/scripts/clear-irqbalance-banned-cpus.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x
+
+# const
+SED="/usr/bin/sed"
+# tunable - overridable for testing purposes
+IRQBALANCE_CONF="${1:-/etc/sysconfig/irqbalance}"
```

Comment:
This assumes the cpu list is at /etc/sysconfig/irqbalance - but I think the location is configurable in cri-o and according to a comment on a PR, the location is different for ubuntu (for example): https://github.com/cri-o/cri-o/pull/4441#discussion_r545234016

Reply:
true, but on RHCOS/RHEL this is where the config file is placed. I'm not aware we support ubuntu as host system.